spec/controllers/application_controller/tags_spec.rb

```diff
@@ -163,8 +163,8 @@ def convert_to_region_id(id)
         expected_options = {
           :model      => "Host",
           :object_ids => [host.id],
-          :add_ids    => [dept_finance.id, env_production.id],
-          :delete_ids => [dept_accounting.id, env_accounting.id]
+          :add_ids    => array_including(dept_finance.id, env_production.id),
+          :delete_ids => array_including(dept_accounting.id, env_accounting.id)
```

Comment:
If the array has more items or less items will this now pass?  Wonder of match_array is a better choice?